examples/dialogflow-webhook-example/main.py

```diff
@@ -0,0 +1,443 @@
+        for user in users:
+            """Example:
+            user_dict: {
+               'user_id': 123456,
+               'first_name': 'Pedro',
+               'last_name': 'Perez',
+               'accounts': {
+                  'checking': {
+                     'transactions': [
+                         {'amount': 50, 'type': 'deposit'},
+                         {'type': 'withdraw', 'amount': '-10'}
+                       ],
+                      'balance': 150
+                    },
+                  'saving': {
+                      'transactions': [
+                          {'amount': 20, 'type': 'deposit'},
+                          {'type': 'deposit', 'amount': 90}
+                        ],
+                      'balance': 110
+                    }
+                }
+             }
+        """
+            user_dict = user.to_dict()
+            return json.dumps({
+                'fulfillmentText':
+                    'From webhook: Here are your account balances. Checking account is {0}  Saving account is {1}. What else can I do for you?'
+                    .format(user_dict['accounts']['checking']['balance'],
+                            user_dict['accounts']['saving']['balance'])
+            })
+
+        return messages.user_id_not_found(user_id)
+
+    def account_transactions_handler(self, **kwargs):
+        """Returns all the transactions (deposits and withdraws)."""
+        flattener = kwargs['flattener']
+        messages = Messages()
+
+        # Get the user_id from the context in the request.
+        user_id = self._get_user_id(flattener)
+
+        # Get the firestore client and extract the user data base on user_id
+        # provided.
+        db = get_firestore_client()
+        client_ref = db.collection(u'users').where(u'user_id', u'==', user_id)
+        users = client_ref.stream()
+        for user in users:
+            """Example:
+            user_dict: {
+               'user_id': 123456,
+               'first_name': 'Pedro',
+               'last_name': 'Perez',
+               'accounts': {
+                   'checking': {
+                     'transactions': [
+                         {'amount': 50, 'type': 'deposit'},
+                         {'type': 'withdraw', 'amount': '-10'}
+                       ],
+                       'balance': 150
+                    },
+                    'saving': {
+                       'transactions': [
+                           {'amount': 20, 'type': 'deposit'},
+                           {'type': 'deposit', 'amount': 90}
+                        ],
+                        'balance': 110
+                     }
+                 }
+             }
+        """
+            user_dict = user.to_dict()
+            message = 'From webhook: Here are all the transactions that I found:\n'
+            all_transactions = ','.join([
+                '{} {}'.format(transaction['type'], transaction['amount'])
```

Comment:
nit: I recommend using f-strings they are [faster](https://realpython.com/python-f-strings/#speed) and more concise / readable.
```suggestion
                f"{transaction['type']} {transaction['amount']}"
```

Reply:
You can use f-strings everywhere you use format in this PR.

I'd only use format if you are leveraging something like truncating or padding to a certain length (e.g. truncate a certain precision after the decimal of a float).